bisqapps/shared/presentation/build.gradle.kts

```diff
@@ -34,25 +38,22 @@ buildConfig {
         buildConfigField("SHARED_LIBS_VERSION", project.version.toString())
         buildConfigField("BUILD_TS", System.currentTimeMillis())
     }
-//    buildConfigField("APP_SECRET", "Z3JhZGxlLWphdmEtYnVpbGRjb25maWctcGx1Z2lu")
-//    buildConfigField<String>("OPTIONAL", null)
-//    buildConfigField("FEATURE_ENABLED", true)
-//    buildConfigField("MAGIC_NUMBERS", intArrayOf(1, 2, 3, 4))
-//    buildConfigField("STRING_LIST", arrayOf("a", "b", "c"))
-//    buildConfigField("MAP", mapOf("a" to 1, "b" to 2))
-//    buildConfigField("FILE", File("aFile"))
-//    buildConfigField("URI", uri("https://example.io"))
-//    buildConfigField("com.github.gmazzo.buildconfig.demos.kts.SomeData", "DATA", "SomeData(\"a\", 1)")
+    //    buildConfigField("APP_SECRET", "Z3JhZGxlLWphdmEtYnVpbGRjb25maWctcGx1Z2lu")
+    //    buildConfigField<String>("OPTIONAL", null)
+    //    buildConfigField("FEATURE_ENABLED", true)
+    //    buildConfigField("MAGIC_NUMBERS", intArrayOf(1, 2, 3, 4))
+    //    buildConfigField("STRING_LIST", arrayOf("a", "b", "c"))
+    //    buildConfigField("MAP", mapOf("a" to 1, "b" to 2))
+    //    buildConfigField("FILE", File("aFile"))
+    //    buildConfigField("URI", uri("https://example.io"))
+    //    buildConfigField("com.github.gmazzo.buildconfig.demos.kts.SomeData", "DATA",
+    // "SomeData(\"a\", 1)")
 
 }
 
 kotlin {
     androidTarget {
-        compilations.all {
-            kotlinOptions {
-                jvmTarget = "1.8"
-            }
-        }
+        compilations.all { kotlinOptions { jvmTarget = "1.8" } }
```

Comment:
I don't think we want this format change, please review indentation / spacing setup